src/hotspot/share/runtime/thread.cpp

```diff
@@ -3589,7 +3589,6 @@ void Threads::add(JavaThread* p, bool force_daemon) {
   p->set_on_thread_list();
 
   _number_of_threads++;
-
```

Comment:
Other nit: empty line L3618 before closing brace doesn't like nice either. I'll leave it to you if you remove it.

Reply:
Yeah I noticed it.